packages/core/src/Account.ts

```diff
@@ -259,7 +259,6 @@ export class Account extends TypedEventEmitter<Events> {
     // we reset the services to re-instantiate a new CryptoClient instance
     await this.initServices(this.apiClient.context);
     const initialPreKeys = await this.service.proteus.createClient(entropyData);
-    await this.service.proteus.initClient(this.storeEngine, this.apiClient.context);
```

Comment:
This was unecessary as it's done in the `this.initClient` at the end (so we were doing it twice)